chart/example-prod.values.yaml

```diff
@@ -0,0 +1,14 @@
+replicaCount: 3
```

Comment:
I dont think the prod files add any values here. The full template already does nearly everything we can do.